src/lightgbm/src/test/scala/VerifyLightGBMClassifier.scala

```diff
@@ -115,6 +117,30 @@ class VerifyLightGBMClassifier extends Benchmarks with EstimatorFuzzing[LightGBM
       train))
   }
 
+  def verifySaveBooster(fileName: String,
+                       outputFileName: String,
+                       labelColumnName: String): Unit = {
+    test("Verify LightGBMClassifier save booster to " + fileName) {
+      // Increment port index
+      portIndex += numPartitions
+      val fileLocation = DatasetUtils.binaryTrainFile(fileName).toString
+      val dataset = readCSV(fileName, fileLocation).repartition(numPartitions)
+      val lgbm = new LightGBMClassifier()
+      val featuresColumn = lgbm.uid + "_features"
+      val featurizer = LightGBMUtils.featurizeData(dataset, labelColumnName, featuresColumn)
+      val rawPredCol = "rawPred"
+      val model = lgbm.setLabelCol(labelColumnName)
+        .setFeaturesCol(featuresColumn)
+        .setRawPredictionCol(rawPredCol)
+        .setDefaultListenPort(LightGBMConstants.defaultLocalListenPort + portIndex)
+        .setNumLeaves(5)
+        .setNumIterations(10)
+        .fit(featurizer.transform(dataset))
+
+      model.saveBooster(session, outputFileName)
```

Comment:
This test will always pass, perhaps theres a way to verify that the output exists/is correct?